boards/posix/nrf52_bsim/Kconfig.defconfig

```diff
@@ -0,0 +1,51 @@
+
+if BOARD_NRF52_BSIM
+
+config BUILD_OUTPUT_BIN
+	default n
```

Comment:
(This one might be okay, as it could override a `default y` from the base definition. Same below.)

Reply:
Yep, it is overriding the misc/Kconfig:225 default y